modules/Core/classes/Integrations/MinecraftIntegration.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+/*
+ *	Made by Partydragen
```

Comment:
same thing about using the `@author` etc stuff here. looks like you got it on some other classes

Reply:
Yea it was not used when i started on it